client/src/components/Login.js

```diff
@@ -85,15 +85,15 @@ export class Login extends Component {
                 <p>Something went wrong! Please check email or password!</p>
               ) : null}
               <Button
-                size="massive"
+                size="huge"
```

Comment:
Hey Blaise, sorry to come back to this again, I think the size can be totally removed here, to make sure the size of the text in the button is consistent with the rest. Thanks :)

Reply:
Done, size removed...